scripts/install_end_to_end_test_dependencies.sh

```diff
@@ -20,6 +20,8 @@ externals=("cmake" "protobuf" "nvm" "node" "foxx")
 
 local_UNIFY=false
 
+install_python
```

Comment:
Shouldn't we just be adding python to the externals list and adding case in the dependency_install_scripts to handle it?